src/coreclr/vm/methodtablebuilder.h

```diff
@@ -958,7 +942,7 @@ class MethodTableBuilder
 
         //-----------------------------------------------------------------------------------------
         // Returns the method type (normal, fcall, etc.) that this type was constructed with.
-        METHOD_TYPE
+        MethodClassification
         GetMethodType() const
```

Comment:
Should this now be renamed `GetMethodClassification()`?

Reply:
Not in this PR. Once this is in, we can clean this up more thoroughly. The `MethodClassification` should also be an `enum class` as opposed to an `enum`.